app/views/home/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+
+<header class="site-header" role="banner">
+  <div class="wrapper">
+      <p class="site-title" >Haxxor News</p>
+			<nav class="site-nav">
+				<div class="trigger">
+					<%= link_to 'Create Post', nil, {class: 'page-link'} %>
+				</div>
+			</nav>
+	</div>
+</header>
```

Comment:
following up on @dbernheisel's note, being clean and consistent with spacing goes a long way towards keeping your code readable and less annoying in the future. looks like you have a handful of extra space chunks in this file